libs/langchain/langchain/retrievers/kendra.py

```diff
@@ -336,13 +354,18 @@ class AmazonKendraRetriever(BaseRetriever):
     page_content_formatter: Callable[[ResultItem], str] = combined_text
     client: Any
     user_context: Optional[Dict] = None
+    score_confidence: float = 0
```

Comment:
This is an attribute fo a pydantic model, can use a Field that with min and max values to limit the value between 0 and 1 on initialization

Reply:
`min_score_confidence: Optional[confloat(ge=0.0, le=1.0)] = None` tried using `confloat` to set limit during initialization but failing during linting `make lint` throwing error `Invalid type comment or annotation  [valid-type]`.